job.go

```diff
@@ -147,3 +148,10 @@ func (j *Job) RunCount() int {
 	runCount := j.runCount
 	return runCount
 }
+// RemoveAfterLastExec update the job in order to remove the job after its last exec
+func (j *Job)RemoveAfterLastExec() *Job {
```

Comment:
could you change this to 
```suggestion
func (j *Job)RemoveAfterLastRun() *Job {
```